### ISSUE TITLE
Please add # of passengers field when setting up alert

### ISSUE THREAD

**Issue by vkhlystun** [samyun/southwest-price-drop-bot#20]
Southwest decreases prices when there are only 1-3 seats left for the flight. This triggers alerts which are useless for family travel.

**Comment by rusd1123** [comment]
I wouldn’t say useless since you could rebook at least 1 ticket at the lower price and keep them under separate PNRs.

**Comment by vkhlystun** [comment]
I agree, it could be useful, but right now I'm getting alerts for the flight that have only 1 seat left. Booking it without ability to put the rest of the family on the same flight makes no sense. I guess I can create 10+ separate alerts for each individual flight this day, but this is too time consuming.
What do you think? Do you plan to add this functionality?

**Comment by flyize** [comment]
Interesting use case.  I only use it to watch travel I've already booked.

**Comment by samyun** [comment]
It should be fairly easy to add a textbox to check for the number of people. I’ve never had to buy flights for multiple people on the same PNR on SW, so I’m not sure how it’d look on their end, number-wise. I’ll investigate it.

**Comment by brussrus** [comment]
Should be easy enough to implement as one of things you pass when you create your URL string is adultPassengersCount=1.  You could default to 1 but have an additional env variable that if set lets you pass a different number of passengers and then just change that portion of the string.
